### ISSUE TITLE
process stopped after 'axial model loaded'

### ISSUE THREAD

**Issue by weiwensydneyUNSW** [Deep-MI/FastSurfer#199]
## Question/Support Request

The process exited unexpectly.

## Screenshots

...
![Screenshot 2022-09-30 142115](https://user-images.githubusercontent.com/50188702/193189759-7fc6d248-bb49-405d-98eb-7a32a3e6c132.png)

## Environment
 - FastSurfer Version: docker - deepmi/fastsurfer:cpu-v1.1.1
 - FreeSurfer Version: ... 
 - OS: WSL2
 - CPU: ...
 - GPU: ...

### Execution

```
docker run --rm --user $(id -u):$(id -g) \
-v /home/wei/1work/fs/test:/data \
-v /home/wei/1work/fs/fastS_an:/output \
-v /home/wei/freesurfer:/fs \
deepmi/fastsurfer:cpu-v1.1.1 \
--t1 /data/sub-1070.nii.gz \
--sd /output \
--sid subject3 \
--fs_license /fs/license.txt \
--parallel
```

Run Command:

**Comment by m-reuter** [comment]
Looks like a bug to me. It conforms your input image but then it loads the original image instead of the conformed? We need to look at this.

**Comment by m-reuter** [comment]
I tried replicating, but for my non-conformed test image it works. It loads the image successfully during which it conforms it, then it runs the three views. So after "Axial model loaded." there is no further output?

**Comment by m-reuter** [comment]
Oh, just now I see that you are using WSL2. We have never tested that and are not supporting it currently. You can try installing a virtual machine with Ubuntu 20:04 and then run a native install or a docker inside that.

**Comment by weiwensydneyUNSW** [comment]
OK thanks Martin.

Wei

**Comment by m-reuter** [comment]
Also see #174 where someone got the CPU version running directly on Windows via docker.

**Comment by xasapakos** [comment]
Same issue here when running the CPU version on MacOs Intel via docker

**Comment by LeHenschel** [comment]
Have you checked that the memory requirements are adjusted? By default docker on Mac only provides 2GB.

See the README or here for details how to do that: https://github.com/Deep-MI/FastSurfer/issues/40#issuecomment-759361297

**Comment by m-reuter** [comment]
Hi @xasapakos @weiwensydneyUNSW  did you get it working? You could also try with our new 2.0 releases. If it still fails we probably need to look at your images to replicate this.

**Comment by weiwensydneyUNSW** [comment]
Hi Martin,

Thank you very much for remembering my (unsolved) question!

No, it didn’t work but I didn’t press on trying it. I will let you know what I see once I run this again.

Wei

**Comment by m-reuter** [comment]
I will close this issue for now. It could be that this issue has been caused by negative values in your input images (such as sometimes in the UK Biobank data). In those cases conform.py failed, which has been fixed in November #210 . 
If your issue still persists, feel free to re-open.
